package.json

```diff
@@ -10,7 +10,7 @@
     "js-yaml": "^3.8.4",
     "json-schema-ref-parser": "^3.1.2",
     "lodash": "^4.16.4",
-    "node-request-by-swagger": "^1.0.6",
+    "node-request-by-swagger": "github:mihaislobozeanu/node-request-by-swagger",
```

Comment:
Is this meant to permanently point to your fork?

Reply:
I've made a small change there ... I don't know another way to point to that change.